iam/terraform/account-specific/bin/deploy-app.sh

```diff
@@ -22,4 +22,4 @@ fi
 set -eo pipefail
 
 terraform init -backend=true -backend-config=bucket="${BUCKET}" -backend-config=key="${KEY}" -backend-config=dynamodb_table="${LOCK_TABLE}" -backend-config=region="${REGION}"
-TF_VAR_my_s3_state_bucket="${BUCKET}" TF_VAR_my_s3_state_key="${KEY}" terraform apply -auto-approve -var "dns_domain=${EFCMS_DOMAIN}"
+TF_VAR_my_s3_state_bucket="${BUCKET}" TF_VAR_my_s3_state_key="${KEY}" terraform apply -auto-approve -var "dns_domain=${EFCMS_DOMAIN}" -var "es_logs_instance_count=${ES_LOGS_INSTANCE_COUNT}" -var "environments=${ENVIRONMENTS}"
```

Comment:
Appears this might be a good opportunity for a little tidying

```
TF_VAR_my_s3_state_bucket="${BUCKET}" 
TF_VAR_my_s3_state_key="${KEY}" 
TF_VAR_dns_domain="${EFCMS_DOMAIN}"
TF_VAR_es_logs_instance_count="${ES_LOGS_INSTANCE_COUNT}"
TF_VAR_environments="${ENVIRONMENTS}"
terraform apply -auto-approve
```